api/v1alpha1/bundleinstance_types.go

```diff
@@ -46,18 +46,28 @@ const (
 type BundleInstanceSpec struct {
 	// ProvisionerClassName sets the name of the provisioner that should reconcile this BundleInstance.
 	ProvisionerClassName string `json:"provisionerClassName"`
+	// Label selector for Bundles resources.
+	Selector *metav1.LabelSelector `json:"selector"`
```

Comment:
Is having the selector mandatory an implication of the implementation? If I just want to create an embedded BI, I would have to provide a label selector as well, is that UX a bit complex? Could we not just create a default selector if one is not provided by the client?

Reply:
Dropped this field in the most recent set of commits.